.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,464 @@
+name: "Synopsys Intelligent Security Scan"
+on:
+  workflow_dispatch:
+   inputs:
+      hpi_name:
+        description: 'Name of hpi file'
+        required: true
+        type: string
+env:
+      XML: com/synopsys/test/integrations/cases/suites/deployment/testng-jenkins.xml
+
+jobs:
+  delivery_stages:
+
+     runs-on: [ linux ]
+
+     steps:
+      
+      - name: unit testing
+        run: |
+          echo "unit testing stage"
+      - name: build
+        run: |
+          echo "build stage"
+      - name: pop
+        run: |
+          echo "Pop stage"
+      - name: install-plugin
+        run: |
+          sudo apt-get update
+          sudo apt-get install -qqy awscli jq openjdk-11-jdk
```

Comment:
@zaman-akib Java 11 required to initiate the e2e execution and nothing to do with Jenkins Java requirement.

Reply:
@zaman-akib Java 11 required to initiate the e2e execution and nothing to do with Jenkins Java requirement.

Reply:
@sig-rnd-int-e2e I got your point but under the hood you are executing e2e Jenkins instance with the Java 11 version. That means your Jenkins instance is using Java 11 when you initiate the e2e Jenkins instance with `java -jar jenkins-cli.jar` command. Currently we are also developing the plugin based on Java 17 since Jenkins is recommending to use Java 17. Therefore, if the e2e test is with Java 11, then the development and testing environment will differ based on the Java version.

Reply:
As per our discussion over call, the changes are fine.